edk2toolext/environment/self_describing_environment.py

```diff
@@ -30,6 +33,9 @@ def __init__(self, workspace_path, scopes=()):
         # Start with the provided set.
         self.scopes = scopes
 
+        # Allow certain directories to be skipped
+        self.skipped_dirs = tuple(map(pathlib.Path, (os.path.join(self.workspace, d) for d in skipped_dirs)))
```

Comment:
should it be only workspace related or should packages path be involved?

Reply:
I was only considering workspace relative paths. Do you think packages path is needed?